changelog/6962.bugfix.rst

```diff
@@ -0,0 +1 @@
+allow caching of parameterized fixtures
```

Comment:
Fixed bug where parametrized fixtures were not being cached correctly, being recreated every time.